codice/ddf
# DDF-5616 Added log sanitization utility and sanitized logs

#### What does this PR do?
Added log sanitization utility and sanitized logs
#### Who is reviewing it? 

@mojogitoverhere @kentmorrissey @brhumphe @Lambeaux @mazarag2 

#### Ask 2 committers to review/merge the PR and tag them here.

@jrnorth
@pklinef

#### How should this be tested?
Full build and install to make sure everything is wired up correctly. Test a log statement that should be sanitized to ensure that the log sanitization utility is working correctly.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: #5616 

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
